tests/cli/commands/test_addraster.py

```diff
@@ -39,4 +39,6 @@ def test_add_raster_to_item(self):
             asset = item.get_assets().get("analytic")
             assert asset is not None
             expected = expected_json("rasterbands.json")
-            assert expected == asset.to_dict().get("raster:bands")
+            self.maxDiff = None
+            for a, b in zip(expected, asset.to_dict().get("raster:bands")):
+                self.assertDictEqual(a, b)
```

Comment:
Not something that needs to be changed in this PR, but this might be a good use-case for [parametrized tests](https://docs.pytest.org/en/6.2.x/parametrize.html) now that we're using `pytest`.

Reply:
Cool, yeah I hadn't looked into that much before. Useful!